webapp/portlet/src/main/webapp/vue-apps/hamburger-menu/components/recent-spaces/SpacePanelHamburgerNavigationItem.vue

```diff
@@ -52,7 +52,7 @@ export default {
   },
   computed: {
     navigationIcon() {
-      return `${this.applicationIcon(this.navigation.icon)} icon-default-color icon-default-size`;
+      return this.navigation?.icon ? `${this.navigation.icon} icon-default-color icon-default-size` : 'fas fa-folder';
```

Comment:
`icon-default-color icon-default-size` has to be used all time for UI coherence IMO.

```suggestion
      return `${this.navigation.icon || 'fas fa-folder'} icon-default-color icon-default-size`;
```

Reply:
fixed